README.md

```diff
@@ -39,26 +41,41 @@ await main()
 ```python
 # OUTPUT
 {
-  '0599010000360091': {
-    'is_rijksmonument': True,
-    'rijksmonument_nummer': '524327',
-    'rijksmonument_url': 'https://monumenten.nl/monument/524327',
-    'is_beschermd_gezicht': False,
-    'beschermd_gezicht_naam': None
+  "0599010000360091": {
+    "is_rijksmonument": true,
+    "rijksmonument_nummer": "524327",
+    "rijksmonument_url": "https://monumentenregister.cultureelerfgoed.nl/monumenten/524327",
+    "is_beschermd_gezicht": false,
+    "beschermd_gezicht_naam": null,
+    "is_gemeentelijk_monument": false,
+    "grondslag": null
   },
-  '0599010000486642': {
-    'is_rijksmonument': False,
-    'rijksmonument_nummer': None,
-    'rijksmonument_url': None,
-    'is_beschermd_gezicht': False,
-    'beschermd_gezicht_naam': None
+  "0599010000486642": {
+    "is_rijksmonument": false,
+    "rijksmonument_nummer": null,
+    "rijksmonument_url": null,
+    "is_beschermd_gezicht": false,
+    "beschermd_gezicht_naam": null,
+    "is_gemeentelijk_monument": false,
+    "grondslag": null
   },
-  '0599010000281115': {
-    'is_rijksmonument': False,
-    'rijksmonument_nummer': None,
-    'rijksmonument_url': None,
-    'is_beschermd_gezicht': True,
-    'beschermd_gezicht_naam': 'Kralingen - Midden'
+  "0599010000281115": {
+    "is_rijksmonument": false,
+    "rijksmonument_nummer": null,
+    "rijksmonument_url": null,
+    "is_beschermd_gezicht": true,
+    "beschermd_gezicht_naam": "Kralingen - Midden",
+    "is_gemeentelijk_monument": false,
+    "grondslag": null
+  },
+  "0599010000146141": {
+    "is_rijksmonument": false,
+    "rijksmonument_nummer": null,
+    "rijksmonument_url": null,
+    "is_beschermd_gezicht": true,
+    "beschermd_gezicht_naam": "Rotterdam - Waterproject",
+    "is_gemeentelijk_monument": true,
+    "grondslag": "Gemeentewet: Aanwijzing gemeentelijk monument (voorbescherming, aanwijzing, afschrift)"
```

Comment:
moeten we `grondslag` hernoemen naar `grondslag_gemeentelijk_monument`?

Reply:
Ja, dat kan!